test/integration/narrow/helpers/other-items-sizes.test.js

```diff
@@ -399,8 +399,8 @@ describe('Other Items Sizes Array Function', () => {
           {
             type: 'MIN_MAX',
             min: 1,
-            max: 9999,
-            error: 'Quantity must be between 1-9999'
+            max: 999999,
+            error: 'Quantity must be between 1-999999'
```

Comment:
![image](https://github.com/DEFRA/ffc-grants-slurry-web/assets/15157025/1d89a7ec-bc18-4a86-bfd1-65f39048e319)
Pumps should be 1-9999